### ISSUE TITLE
Map Snapshot functionality is not consistently labelled

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#1423]
> Note also that screenshot is still used in the UI. Specifically, in the Game History panel context menu, the command label is Save Screenshot (but it runs the exact same code as the main menu's Export > Export Map Snapshot command). It's possible that this label was simply overlooked as part of 0e86423. @DanVanAtta or @ron-murhammer, please advise if that label should be updated to use snapshot as part of this PR.

From: https://github.com/triplea-game/triplea/pull/1419

yes, the above is a bug, should be consistent "Export Map Snapshot "

**Comment by ssoloff** [comment]
@DanVanAtta I didn't see a commit associated with this issue, so I wanted to clarify something just in case there was some confusion.

The change from "screenshot" to "map snapshot" I made in #1419 for which I asked for review in this [comment](https://github.com/triplea-game/triplea/pull/1419#discussion_r94848335) was for the file chooser extension description, which is not the same change we discussed making in this [comment](https://github.com/triplea-game/triplea/pull/1419#issuecomment-269931679), which, I believe, was the reason for creating this ticket.

To be clear, the issue referenced at the top of this ticket is the fact that the Game History panel context menu contains a command labeled "Save Screenshot" instead of "Export Map Snapshot".  That has not been changed in #1419 based on your previous [feedback](https://github.com/triplea-game/triplea/pull/1419#issuecomment-270062266).

Apologies if lead you to believe that I had updated the command label in the Game History panel context menu as part of #1419.
